### PR TITLE
Other timings

### PR DESCRIPTION
**Pull request by F2had**
- Show Other important timings (Midnight, First and Last Third, Imsak)

### CODE REVIEW

File: lib/modules/prayer_times/models/prayer_times.dart [hunk 1]
```diff
@@ -21,32 +22,55 @@ Map<PrayerType, DateTime> _prayerTimesMapFromJson(Map<String, dynamic> json) {
...
+  });
+  return otherTimings;
+}
+
 class DayPrayerTimes {
   final Map<PrayerType, DateTime> prayerTimes;
   final DateTime date;
   final CalculationMethod calculationMethod;
   final String timeZoneName;
-  // TODO: add hijri date
```
Comment (Amjad50) [inline lib/modules/prayer_times/models/prayer_times.dart:29]: was this removed by mistake?

Comment (Amjad50) [reply lib/modules/prayer_times/models/prayer_times.dart:29]: beside that, its good.

**Review by Amjad50 — APPROVED** [review]

File: lib/modules/prayer_times/models/prayer_times.dart (hunk 1 above)
Comment (F2had) [reply lib/modules/prayer_times/models/prayer_times.dart:29]: We already have hijri date

Comment (Amjad50) [reply lib/modules/prayer_times/models/prayer_times.dart:29]: oops sorry

**Pull request merged**